fmriprep/__init__.py

```diff
@@ -23,6 +23,7 @@
     __description__,
     __longdesc__
 )
+from .due import due, Url
```

Comment:
Woops. Forgot to import `Doi`.

Reply:
thanks! pushed the fix.  btw -- feel welcome to push those tiny fixes directly into my fork/branch -- all devs of fmriprep should have permission for that